notebooks/samples/104 - Train, Test, Evaluate for Regression with Auto Imports Dataset.ipynb

```diff
@@ -0,0 +1,329 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 104 - Train, Test, Evaluate for Regression with Auto Imports Dataset\n",
+    "\n",
+    "This sample notebook is based on the Gallery \"Sample 6: Train, Test, Evaluate for Regression: Auto Imports Dataset\" for AzureML Studio. This experiment demonstrates how to build a regression model to predict the automobile's price. The process includes training, testing, and evaluating the model on the Autom Imports data set.\n",
+    "\n",
+    "This sample demonstrates the use of several members of the mmlspark library:\n",
+    "- TrainRegressor\n",
+    "- SummarizeData\n",
+    "- CleanMissingData\n",
+    "- ComputeStatistics\n",
+    "- FindBestModel"
```

Comment:
Please use something like:
- ``TrainRegressor``
- ``FindBestModel``
and add link to the documentation for each of these functions.